test/tests_for_integration/api_to_db_test.py

```diff
@@ -1580,12 +1607,22 @@ def test_run_job_batch_fail_no_workspace_access_for_batch(ee2_port, ws_controlle
     _run_job_batch_fail(ee2_port, TOKEN_NO_ADMIN, params, {"wsid": 2}, err)
 
 
-def test_run_job_batch_fail_no_workspace_access_for_job(ee2_port):
+def test_run_job_batch_fail_no_allowed_wsid(ee2_port):
     params = [
         {"method": _MOD},
         {"method": _MOD, "wsid": 1},
     ]
     # this error could probably use some cleanup
+    err = InvalidParameterForBatch().__doc__
```

Comment:
I would recommend using the actual error string here. If I fatfinger an unintended change to the error then this test won't catch it, but if I use the bare string it will.

Reply:
ok

Reply:
:+1: